docs/data-sources/ip_address.md

```diff
@@ -0,0 +1,43 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud_ip_address Data Source - terraform-provider-anxcloud"
+subcategory: ""
+description: |-
+  Retrieves an IP address.
+---
+
+# anxcloud_ip_address (Data Source)
+
+Retrieves an IP address.
+
+### Known limitations
+
+- Only unique IP addresses can be retrieved.
```

Comment:
I think we need to expand a little on how 'uniqueness' is determined in this context, I assume this is based on the Engine context? So RFC 1918 IP addresses work fine, as long as they only exist once within the access scope of the Engine token used? From the Engine PoV ever 'IP address' object is unique, so we need to use concise language here.

Reply:
Something like this?
> Only IP addresses unique to the scope of your access token for Anexia Cloud can be retrieved.

Reply:
Sounds good, but now I have another questions :sweat_smile:
Does this limitation apply to all retrieval styles? Can't I retrieve any address object via its identifier, isn't just the `address` search limited?

Reply:
This should cover the limitation:
> When using the address argument, only IP addresses unique to the scope of your access token for Anexia Cloud can be retrieved. You can however get a unique result by specifying the related VLAN or network prefix.

Reply:
Alright, that is much more helpful I think :)